lib/services/feature_flag_service.dart

```diff
@@ -11,35 +13,52 @@ class FeatureFlagService {
   static final FeatureFlagService instance =
       FeatureFlagService._privateConstructor();
   static const kBooleanFeatureFlagsKey = "feature_flags_key";
+  FeatureFlags defaultFlags = FeatureFlags(
```

Comment:
Maybe this can be a static const on FeatureFlags class itself? e.g. `FeatureFlags.default` or `FeatureFlags.defaultFlags`.